src/server/capacity_unit_calculator.cpp

```diff
@@ -122,21 +134,25 @@ void capacity_unit_calculator::add_multi_get_cu(int32_t status,
     }
 
     if (status == rocksdb::Status::kNotFound) {
+        _read_hotkey_collector->capture_hash_key(hash_key, 1);
```

Comment:
I notice that you try to capture not-found key for multi_get, but not for single get, could you please explain why you design it?